README.md

```diff
@@ -30,7 +30,7 @@ Component Authors need to build a C#/WinRT projection for .NET5+ targets.
 
 [WinUI3.0](https://github.com/Microsoft/microsoft-ui-xaml) is the effort to lift official native Microsoft UI controls and features out of the operating system, so app developers can use the latest controls and visuals on any in-market version of the OS. C#/WinRT is needed to support the changes required for lifting the XAML APIs out of Windows.UI.XAML and into Microsoft.UI.XAML.
 
-However, C#/WinRT is a general effort and is intended to support other scenarios and versions of the .NET runtime, compatible down to .NET Standard 2.0.
+However, C#/WinRT is a general effort and is intended to support other scenarios such as building custom projections and authoring C# components.
```

Comment:
custom projections and authoring aren't really 'other scenarios' - those are mainline C#/WinRT features.  The intent of the original sentence was to say that with netstandard2 compatibility, C#/WinRT could support WinML, Unity, etc.